tests/app/legacy/v2/notifications/test_utils.py

```diff
@@ -63,10 +63,10 @@ async def test_send_push_notification_helper_logs_exception(
         mock_template = AsyncMock(spec=Template)
         mock_template.build_message.return_value = 'test_message'
         mock_provider = AsyncMock(spec=ProviderAWS)
-        mock_provider.send_notification.side_effect = ProviderNonRetryableError
-        personalisation: dict[str, str | int | float] = {'name': 'John'}
+        mock_provider.send_notification.side_effect = NonRetryableError
+        personalization: dict[str, str | int | float] = {'name': 'John'}
 
-        await send_push_notification_helper(personalisation, '12345', mock_template, mock_provider)
+        await send_push_notification_helper(personalization, '12345', mock_template, mock_provider)
```

Comment:
super nit: are we using `personalization` or `personalisation`  in this repo? I understand this is just for a test - so I suppose it does not matter

Reply:
we have to match the current notification-api. we were going to run V3 with the American spelling, but that is a long ways off. `personlisation` is correct here.

Reply:
I don't know.  Both were present when I needed to fix a merge conflict, and I went with Z (:eagle: :us:).

Reply:
I changed it to S because that's used elsewhere in the file.

Reply:
I went with "s" for consistency with notification-api and the spelling that will be returned in error messages. I think we want to change the spelling for v3, but I thought it would be good to stay consistent (use "s") in the legacy files.